<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="espaceRVB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Espace RVB</string>
    </property>
    <widget class="QPushButton" name="parcourir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>parcourir</string>
     </property>
    </widget>
    <widget class="QLabel" name="labOrg">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image original</string>
     </property>
    </widget>
    <widget class="QLabel" name="compB">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>40</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labBleue">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Composante bleue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficheB">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>240</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher le canal bleu</string>
     </property>
    </widget>
    <widget class="QLabel" name="labVerte">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Composante verte</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficheV">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>240</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher le canal vert</string>
     </property>
    </widget>
    <widget class="QLabel" name="compV">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="afficheR">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>240</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher le canal rouge</string>
     </property>
    </widget>
    <widget class="QLabel" name="compR">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labRouge">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Composante rouge</string>
     </property>
    </widget>
    <widget class="QLabel" name="original">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="dimension">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labDim">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dimention</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Nvespcolor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Nouvelle espace de couleurs</string>
    </property>
    <widget class="QRadioButton" name="rbYUV">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>YUV</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbYCC">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>YCbCr</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbHSV">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV</string>
     </property>
    </widget>
    <widget class="QPushButton" name="valid">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLabel" name="labNvImg">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Nvimg">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouvelle image</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="contr">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>411</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Contours</string>
    </property>
    <widget class="QPushButton" name="cntrValid">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLabel" name="seuil1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Seuil 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="seuil2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Seuil 2</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtS1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtS2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="imgCntr">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Contours</string>
     </property>
    </widget>
    <widget class="QLabel" name="labImgCntr">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTP1">
    <property name="title">
     <string>TP1</string>
    </property>
   </widget>
   <addaction name="menuTP1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
